src/dialogs/more-info-dialog.html

```diff
@@ -145,43 +131,18 @@ <h2>
     return hass.states[hass.moreInfoEntityId] || null;
   }
 
-  /**
-   * We depend on a delayed dialogOpen value to tell the chart component
-   * that the data is there. Otherwise the chart component will render
-   * before the dialog is attached to the screen and is unable to determine
-   * graph size resulting in scroll bars.
-   */
-  computeIsLoadingHistoryData(delayedDialogOpen, stateHistoryLoading) {
-    return !delayedDialogOpen || stateHistoryLoading;
-  }
-
-  computeHasHistoryComponent(hass) {
-    return window.hassUtil.isComponentLoaded(hass, 'history');
-  }
-
-  computeShowHistoryComponent(hasHistoryComponent, stateObj) {
-    return this.hasHistoryComponent && stateObj &&
-      window.hassUtil.DOMAINS_WITH_NO_HISTORY
-        .indexOf(window.hassUtil.computeDomain(stateObj)) === -1;
-  }
-
   stateObjChanged(newVal) {
     if (!newVal) {
       this.dialogOpen = false;
+      this._showSettings = false;
       return;
     }
 
-    window.setTimeout(() => {
-      // allow dialog to render content before showing it so it is
+    requestAnimationFrame(() => requestAnimationFrame(() => {
```

Comment:
Why delay 2 frames?

Reply:
So this will guarantee that it is done after the next paint. Using ms is not guaranteed to do that. 

I learned something new 😉 https://www.youtube.com/watch?v=cCOL7MC4Pl0